docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -36,34 +36,70 @@ Vivarium - CSU - Simulating Multiple Myeloma registries
 +------------------------------------+
 | List of abbreviations              |
 +=======+============================+
-|       |                            |
+|   MM  | Multiple myeloma           |
++-------+----------------------------+
+| RRMM  | Relapsed/ Refractory MM    |
++-------+----------------------------+
+|   isa.| Isatuximab                 |
 +-------+----------------------------+
 
+
 .. _1.0:
 
 1.0 Background
 ++++++++++++++
+For this project, we are interested in the expected impact of Isatuximab on the overall Unites States population, among a registry population, and in key-subpopulations listed below, in comparison to a business as usual treatment scenario over 5 years.
```

Comment:
I wonder if we should make a note about the sub-aim for this project that Bill discussed of the "power calculation" type approach of how many patients would need to be in a registry population in order to observe a significant effect of a given magnitude, and how many patients per year out of all patients per year would need to be enrolled over X years in order to reach that target.

Reply:
That's a great note, thanks! I have added that in the 'Modeling aims and objectives' section.